unstructured/chunking/title.py

```diff
@@ -9,11 +10,80 @@
     Element,
     ElementMetadata,
     Table,
+    TableChunk,
     Text,
     Title,
 )
 
 
+def chunk_by_characters(
+    elements: List[Element],
+    num_characters: Optional[int] = 200,
+) -> List[Element]:
+    """Uses num_characters to return elements within a document in specific sizes.
+    Currently only applies to Table elements.
+     Parameters
+    ----------
+    elements
+        A list of unstructured elements. Usually the output of a partition functions.
+    num_characters
+        Splits elements text field so that a section matches the desired length."""
+
+    chunked_elements: List[Element] = []
+
+    for element in elements:
+        if isinstance(element, Table):
+            char_len = len(element.text)
+            html_table = element.to_dict().get("text_as_html")
```

Comment:
```suggestion
            html_table = getattr(element, "text_as_html")
```

Reply:
shoot - I remember why I didn't do this:

```
>               html_table = getattr(element, "text_as_html")
E               AttributeError: 'Table' object has no attribute 'text_as_html'
```

I think I need to do the same thing as I did below in the actual chunking function:
`    text_as_html = element.text_as_html if hasattr(element, "text_as_html") else None`